### DISCUSSION TITLE
Have you planed on the release date for the next official version?

### DISCUSSION THREAD

**Discussion by norikoji1** [airsdk/Adobe-Runtime-Support#1727] [Q&A]
Hi, I have some question.

It's not issue, so I posted here.
Should I post which forum when ask questions?

Today, I packaged our application with AIR SDK 33.1.1.743.
But the crash occurred only Macs with M1 chips when displayed webkit page.
There were no problem other CPU(ex.Intel)'s Mac PC.

I've remembered to get answer about issue of #1660 few days ago.
----
Note that we're also taking away the WebKit library ... as it's only available for x86_64 and not on M1 Apple silicon; so then the HTMLoader/StageWebView functionality would be provided by the operating system's "webview" component and none of these (with the exception of the Windows IE-based/MSHTML one) supports plug-ins...
----

 I will try to package our application with AIR SDK 33.1.1.758 tomorrow.
Even if we can confirm that it was not crash, AIR SDK 33.1.1.758 is not officially released at this time.
So we can't use that SDK to release our application.

we would like to ask you about the upcoming official release.
do you have some release plans?

thank you

**Comment by ajwfrost** [comment]
Hi

The next release is expected in around 7-10 days - hopefully the first half of next week but there's one issue we're trying to resolve regarding the new Gradle-based build system that needs to go into it.

If you're able to verify that your app still works with the HTML engine from the OS, rather than the built-in webkit - by using 33.1.1.758 or 33.1.1.779 per your above comment - then it would be good to know, since the bulk of the changes we're making currently are on Android so your behaviour on M1 silicon should be pretty similar between 33.1.1.758 and the next official release..

thanks

**Comment by 2jfw** [comment]
I saw that Harman _now_ consequently pushes out new releases approx. like every 14 days - Much appreciated 👍

**Reply by ajwfrost** [reply]
TBH I'd prefer it if we didn't! We keep talking about settling into a regular rhythm with maybe minor releases every month and a more major update every 3 months. But things keep moving a little more quickly and we want to get fixes out into people's hands as soon as possible.... we're working on a simpler (and lower risk, and lower cost) way of making updates available though which should also simplify some of this and free us up for the bigger changes....
